File: ShoppingList/src/main/java/org/openintents/shopping/ui/dialog/NewListDialog.java
package org.openintents.shopping.ui.dialog;

import android.content.Context;

import org.openintents.shopping.R;

public class NewListDialog extends RenameListDialog {

    public NewListDialog(Context context) {
        super(context);

        setTitle(R.string.ask_new_list);
    }

    public NewListDialog(Context context, DialogActionListener listener) {
        super(context);

        setTitle(R.string.ask_new_list);
        setDialogActionListener(listener);
    }

}

File: ShoppingList/src/main/java/org/openintents/shopping/ui/dialog/NewListDialog.kt
package org.openintents.shopping.ui.dialog

import android.content.Context
import org.openintents.shopping.R

class NewListDialog : RenameListDialog {

    constructor(context: Context) : super(context) {
        setTitle(R.string.ask_new_list)
    }

    constructor(context: Context, listener: DialogActionListener) : super(context) {
        setTitle(R.string.ask_new_list)
        setDialogActionListener(listener)
    }
}
